vault/rollback.go

```diff
@@ -134,20 +179,28 @@ func (m *RollbackManager) triggerRollbacks() {
 		fullPath := e.namespace.Path + path
 
 		// Start a rollback if necessary
-		m.startOrLookupRollback(ctx, fullPath, true)
+		if firstRun {
+			wg.Add(1)
+			m.startOrLookupRollback(ctx, fullPath, true, jobs)
+		} else {
+			m.startOrLookupRollback(ctx, fullPath, true, nil)
+		}
+	}
+	if firstRun {
+		close(jobs)
+		wg.Wait()
 	}
 }
 
 // startOrLookupRollback is used to start an async rollback attempt.
 // This must be called with the inflightLock held.
```

Comment:
This is wrong, isn't it? We grab the inflightLock within the function, it doesn't need to be held by the caller.

Reply:
That commeent was from Armon in 2015.  But yeah, the comment is inaccurate